blocks/products/build/block.json

```diff
@@ -14,12 +14,16 @@
 	"attributes": {
 		"shopCatalogApiUrlBase": {
 			"type": "string",
-			"default": "https://shopcatalog.com/wp-json/wc/v3/products/"
+			"default": "https://shopcatalog-com-develop.go-vip.net/wp-json/wc/v3/products/"
 		},
```

Comment:
@douglas-johnson Should this be the live shop catalog URL instead of the development site?

Reply:
Yes, thanks for reminding me. I thought I discarded that.

Reply:
Oh, I did discard it, but I forgot to build it. Just pushed.